wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.19.0/1__userdevfile.sql

```diff
@@ -0,0 +1,24 @@
+--
+-- Copyright (c) 2012-2020 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+
+-- add userdevfile table
+CREATE TABLE userdevfile (
+    id             VARCHAR(255)    NOT NULL UNIQUE,
+    devfile_id     BIGINT          NOT NULL UNIQUE,
+    generated_name VARCHAR(255) ,
+    PRIMARY KEY (id)
+);
+CREATE INDEX index_userdevfile_devfile_id ON userdevfile (devfile_id);
+CREATE INDEX index_userdevfile_generated_name ON userdevfile(generated_name);
+ALTER TABLE userdevfile ADD CONSTRAINT fk_userdevfile_devfile_id FOREIGN KEY (devfile_id) REFERENCES devfile (id);
```

Comment:
Please consider placing it after `CREATE INDEX index_userdevfile_devfile_id` or after all indexes.

Currently, it's in really not a suitable place between name and generate name.